models/balancer/avalanche_c/balancer_v2_avalanche_c_liquidity.sql

```diff
@@ -1,236 +1,21 @@
+
+{% set blockchain = 'avalanche_c' %}
+
 {{
     config(
-        schema='balancer_v2_avalanche_c',
+        schema='balancer_v2_' + blockchain,
         alias = 'liquidity',
-        
         materialized = 'table',
         file_format = 'delta',
-        post_hook='{{ expose_spells(\'["avalanche_c"]\',
-                                    "project",
-                                    "balancer_v2",
-                                    \'["stefenon", "viniabussafi"]\') }}'
+        post_hook='{{ expose_spells(\'["{{blockchain}}"]\',
```

Comment:
In the arb v2 liq spell you have `expose_spells(\'["arbitrum"]\',` 
Just noting here so can change one of them to match the other.